ais_service_discovery/call_service.py

```diff
@@ -6,10 +7,18 @@
 from .sqs import SqsAdaptor, Send
 from json import dumps
 
-function = client('lambda')
-sns = client('sns')
-sqs = client('sqs')
-service_discovery = client('servicediscovery')
+from botocore.config import Config
+
+
+BOTO_MAX_ATTEMPTS = int(env.get('BOTO_MAX_ATTEMPTS', 10))
+BOTO_READ_TIMEOUT = float(env.get('BOTO_READ_TIMEOUT', 300))
```

Comment:
Should the default values be set that high?
IMO they should be kept low, given how they can be overridden from the project that uses them.

Reply:
I was thinking in terms of our current issue, but I could decrease them and we just configure it high. I'll do that